postgres/tests/test_replication_slot.py

```diff
@@ -19,7 +19,7 @@ def test_physical_replication_slots(aggregator, integration_check, pg_instance):
     check = integration_check(pg_instance)
     # It seemingly can take a small amount of time for the pg_replication_slots to be saturated
     # TODO: Poll for its existence as a ready check
-    time.sleep(1)
+    time.sleep(5)
```

Comment:
curious about this change? did it fix the negative stat you posted about in slack?

Reply:
This was for a different race condition where the replication didn't always make it in one second but seems to always make it in five